content/en/blog/_posts/2021-11-18-persistentvolume-reclaim-policy-improvements.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: blog
+title: "Kubernetes 1.23 PersistentVolume reclaim policy improvements"
+date: 2021-11-18T10:00:00-08:00
+slug: kubernetes-1-23-persistentvolume-reclaim-policy-improvements
+---
+
+**Author:** Deepak Kinni (VMware)
+
+[PersistentVolume](/docs/concepts/storage/persistent-volumes/) (or PVs for short) are
+associated with [Reclaim Policy](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaim-policy).
+The Reclaim Policy is used to determine the actions that need to be taken by the storage
+backend on deletion of the PV.
+Where the reclaim policy is `Delete`, the expectation is that the storage backend
+releases the storage resource that was allocated for the PV. In essence, the reclaim
+policy needs to honored on PV deletion.
+
+With the recent Kubernetes v1.23 release, an alpha feature lets you configure your
+cluster to behave that way and honor the configured reclaim policy.
+
+
+## How did reclaim work in previous Kubernetes releases?
+
+[PersistentVolumeClaim](/docs/concepts/storage/persistent-volumes/#Introduction) (or PVC for short) is
+a request for storage by a user. A PV and PVC are considered [Bound](/docs/concepts/storage/persistent-volumes/#Binding)
+if there is a newly created PV or a matching PV is found. The PVs themselves are
+backed by a volume allocated by the storage backend. 
+
+Normally, if the volume is to be deleted, then the expectation is to delete the
+PVC for a bound PV-PVC pair. However, there are no restrictions to delete a PV
+prior to deleting a PVC.
+
+First, I'll demonstrate the behavior for clusters that are running an older version of Kubernetes.
+
+#### Create PVC and wait for it to get bound
+
+Create the PVC defined by a `pvc.yaml`
+```
+kubectl create -f pvc.yaml 
+```
+The following output is seen when the PVC is created:
+```
+persistentvolumeclaim/example-vanilla-block-pvc created
+```
```

Comment:
I don't know if it is useful to show lines 37-43 as the content of pvc.yaml is not displayed.  It's probably sufficient to just show an existing bound PVC/PV.